packages/webapp-libs/webapp-finances/src/config/routes.ts

```diff
@@ -1,14 +1,15 @@
 import { nestedPath } from '@sb/webapp-core/utils';
 
 export const RoutesConfig = {
-  finances: nestedPath('finances', {
+  currentSubscriptions: nestedPath('subscriptions/current-subscriptions', {
```

Comment:
`'subscriptions/current-subscription'` instead of `'subscriptions/current-subscriptions'`